### PR TITLE
Basic Initialize

### PR DESCRIPTION
**Pull request by mrproliu**
* Add basic workflow
* Add Code lint check
* Add dist licenses and notice
* Add CHANGES.md

### CODE REVIEW

**Review by dmsolr — APPROVED** [review]

File: dist/LICENSE [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+=======================================================================
+Apache SkyWalking Rover Subcomponents:
+
+Apache SkyWalking Rover project contains subcomponents with separate copyright
+notices and license terms. Your use of the source code for the these
+subcomponents is subject to the terms and conditions of the following
+licenses.
+
+========================================================================
```
Comment (wu-sheng) [inline dist/LICENSE:211]: I think the following are not necessary.

**Comment by wu-sheng** [comment]
Add asf merge control file, please.

File: dist/LICENSE (hunk 1 above)
Comment (mrproliu) [reply dist/LICENSE]: Removed.

**Comment by mrproliu** [comment]
> Add asf merge control file, please.

Have been added. Just some configs need to be updated.

**Pull request merged**